src/Shared/AuthButton/AuthButton.tsx

```diff
@@ -0,0 +1,15 @@
+interface Prop{
+    text:string
+}
+
+export default function AuthButton(props:Prop) {
+  return (
+    <button
+    type="submit"
+    className="bg-slate-50 transition duration-100 hover:bg-gray-800  text-slate-950  hover:text-slate-50  rounded-lg px-4 py-2 mt-2 font-medium "
+  >
+    {props.text}
```

Comment:
You can make it as a regular button in which we pass the text between the element:
`<AuthButton>Button Text</AuthButton>` and the text will be provided to the children prop which you can get in the component as follows:
```export default function AuthButton({children}){
<button>{children}</button>
} ```